js/index.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [Optional] Please remove the comment and take care of the linter error to have a much cleaner code.